src/DiseaseStatus.H

```diff
@@ -25,6 +25,7 @@ void assignHospital (const int a_i,       /*!< agent index */
                      int* const a_hosp_i, /*!< hospital index i */
                      int* const a_hosp_j, /*!< hospital index j */
                      const PTDType& a_ptd /*!< Particle tile data */) {
+    AMREX_ALWAYS_ASSERT(!inHospital(a_i, a_ptd));
```

Comment:
Does this need to be `ALWAYS_ASSERT`, or would `ASSERT` suffice?

Reply:
I guess `ASSERT` will suffice; failure of this assertion will always be due to a coding error, not a runtime error.